.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on: [push, pull_request]
+
+env:
+  DEFAULT_COMPOSER_FLAGS: "--prefer-dist --no-interaction --ignore-platform-reqs"
```

Comment:
The `--ignore-platform-reqs` here is the reason why your builds are failing. Instead of installing the right PHPUnit version for the PHP version on which you are running the tests, Composer will now install the highest version allowed as can be seen [here](https://github.com/luyadev/luya-testsuite/pull/39/checks?check_run_id=1485397698#step:4:43) and [here](https://github.com/luyadev/luya-testsuite/pull/39/checks?check_run_id=1485397636#step:4:43)

Removing the `--ignore-platform-reqs` should solve your issue / https://github.com/Yoast/PHPUnit-Polyfills/issues/7

Reply:
yes, sure how could i overlook that. Thanks for pointing me into the right directions :+1:

Reply:
yvw